FPSCamera/InstanceCameras/VehicleCamera.cs

```diff
@@ -173,8 +173,9 @@ public void GetInstanceSpeed(Vector3 position)
 
         public void GetLastStopExchange()
         {
-            int lastGone = ImprovedPublicTransport2.Detour.VehicleManagerMod.m_cachedVehicleData[(int)followInstance].LastStopGonePassengers;
-            int lastNew = ImprovedPublicTransport2.Detour.VehicleManagerMod.m_cachedVehicleData[(int)followInstance].LastStopNewPassengers;
+            ushort firstVehicle = vManager.m_vehicles.m_buffer[(int)followInstance].GetFirstVehicle(followInstance);
```

Comment:
Does IPT2 only cache the first vehicle for trains? I noticed that the vanilla game will [leave a train reversed when going the other way](https://github.com/tony56a/Skylines-FPSCamera/blob/5399081a229ff7f5dfed933be3ba15e98962cf3a/FPSCamera/InstanceCameras/VehicleCamera.cs#L58)

Reply:
Apparently. Before I added the ```firstVehicle``` it always showed ```-0 | +0``` when reversed. But now it works.

Reply:
Sounds good if this was verified for trains. I'll apply the changes and publish the mod in a few hours then.